repositories/database_methods.py

```diff
@@ -209,4 +209,49 @@ def edit_post(post_id: int, post_content: str) -> bool:
                             WHERE
                                 post_id = %s
                             ''', [post_content, post_id])
-            # what can I return here?
\ No newline at end of file
+            # what can I return here?
+
+def get_user_concentration(user_id: int) -> dict[str, Any] | None:
+    pool = get_pool()
+    with pool.connection() as connection:
+        with connection.cursor(row_factory=dict_row) as cursor:
+            cursor.execute('''
+                            SELECT 
+                                concentration
+                            FROM 
+                                users
+                            WHERE 
+                                user_id = %s
+                            ''', [user_id])
+            concentration = cursor.fetchone()
+            return concentration
+        
+def get_user_first_name(user_id: int) -> dict[str, Any] | None:
+    pool = get_pool()
+    with pool.connection() as connection:
+        with connection.cursor(row_factory=dict_row) as cursor:
+            cursor.execute('''
+                            SELECT 
+                                first_name
+                            FROM 
+                                users
+                            WHERE 
+                                user_id = %s
+                            ''', [user_id])
+            first_name = cursor.fetchone()
+            return first_name
+
+def get_user_last_name(user_id: int) -> dict[str, Any] | None:
+    pool = get_pool()
+    with pool.connection() as connection:
+        with connection.cursor(row_factory=dict_row) as cursor:
+            cursor.execute('''
+                            SELECT 
+                                last_name
+                            FROM 
+                                users
+                            WHERE 
+                                user_id = %s
+                            ''', [user_id])
+            last_name = cursor.fetchone()
+            return last_name
```

Comment:
You could probably just combine the "get_user_first_name" and "get_user_last_name" into one and return it altogether as "get_user_full_name_by_id". Another option would be to get those user's information in one single method and just use that ("get_user_information_by_id") and because you are using it for profile, it should be okay!

Reply:
Also adding concentration into the suggested method!! I forgot to say

Reply:
perfect